server.js

```diff
@@ -43,97 +40,90 @@ app.get('/', function homepage(req, res) {
-  // It would be seriously overkill to save any of this to your database.
-  // But you should change almost every line of this response.
   res.json({
     woopsIForgotToDocumentAllMyEndpoints: false, // CHANGE ME ;)
     message: "Welcome to my personal api! Here's what you need to know!",
     documentationUrl: "https://github.com/SwethaMuralidharan/express-personal-api/blob/master/README.md", // CHANGE ME
     baseUrl: "https://enigmatic-caverns-31537.herokuapp.com/", // CHANGE ME
     endpoints: [
-      {method: "GET", path: "/api", description: "Describes all available endpoints"}, // done
-      {method: "GET", path: "/api/profile", description: "Data about me"}, // done
-
-      {method: "GET", path: "/api/profile/projects", decsription: "Describes all my projects."}, // done
-      {method: "GET", path: "/api/profile/:profile_id/projects/:_id",description: "displays particular project info"}, //done
-      {method: "POST",path: "/api/profile/:profile_id/projects",description:"creates new project"}, //done
-      {method: "DELETE", path:"/api/profile/:profile_id/projects/:_id", description:"deletes that project"},//done
-      {method: "PUT", path:"/api/profile/:profile_id/projects/:_id", description:"Updates that project by ID"},//done
-
-      {method: "GET", path: "/api/profile/:profile_id/vacation",description:"destinations i've visited"},//done
-      {method: "POST",path:"/api/profile/:profile_id/vacation",description:"Creates an entry for vacations under planning"}
-
+      { 
+        method: "GET", 
+        path: "/api", 
+        description: "Describes all available endpoints"
+      },
+      { 
+        method: "GET", 
+        path: "/api/profile", 
+        description: "Data about me"
+      },
+
+      { 
+        method: "POST",
+        path: "/api/profile/:profile_id/projects",
+        description:"creates new project"
+      },
+      { 
+        method: "DELETE", 
+        path:"/api/profile/:profile_id/projects/:_id", 
+        description:"deletes that project"
+      },
+      { 
+        method: "PUT", 
+        path:"/api/profile/:profile_id/projects/:_id", 
+        description:"Updates that project by ID"
+      },
+
+      { 
+        method: "GET", 
+        path: "/api/profile/:profile_id/vacation",
+        description:"destinations i've visited"
+      },
+      { 
+        method: "POST",
+        path:"/api/profile/:profile_id/vacation",
+        description:"Creates an entry for vacations under planning"
+      }
     ]
   })
 });
 
-/*End Point 2 - Get Profile Info*/
 app.get('/api/profile', function show(req, res) {
-  console.log("inside get profile",req.body);
   db.Profile.find({},function(err,new_profile){
     if(err) { console.log(" new profile not exist", err)}
     res.json(new_profile);
   })
 });
 
-/* End point 3 - Get All Projects */
-app.get('/api/profile/projects', function(req,res){
-  console.log("inside get projects", req);
-  db.Profile.find({},function(err,projects){
```

Comment:
This does NOT get all projects. It gets all profiles. Your callback argument should be `profiles`, not `projects`.